template/src/steps/fetchAccounts.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  createIntegrationEntity,
+} from '@jupiterone/integration-sdk';
+
+const step: IntegrationStep = {
+  id: 'fetch-accounts',
+  name: 'Fetch accounts',
+  types: ['my-integration-account'],
```

Comment:
We always use the form `my_integration_account`. I think we'll want to encourage that for consistency. Sorry if I've been missing that in all the other codes!

Reply:
ah biig oops. I think it's been consistent in other places.